src/components/devices/sma_sunny_boy/bat_smart_energy.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+	<div class="device-sunnyboy-bat">
+		<openwb-base-heading>
+			Einstellungen für SMA Sunny Boy Batteriespeicher
+			<span class="small">(Modul: {{ $options.name }})</span>
+		</openwb-base-heading>
+		<openwb-base-alert subtype="info">
+			Diese Komponente benötigt keine Einstellungen.
+		</openwb-base-alert>
+	</div>
+</template>
+
+<script>
+export default {
+	name: "DeviceSunnyBoyBat",
```

Comment:
Der Name des Moduls wird bereits in `bat.vue` genutzt und sollte eindeutig sein. Selbes gilt für die Klasse des ersten DIV im Template.
```suggestion
	name: "DeviceSunnyBoyBatSmartEnergy",
```